### ISSUE TITLE
FPC version 2.6.2 image

### ISSUE THREAD

**Issue by deniscostadsc** [kveroneau/fpc-docker#1]
Hi, I have a application that needs version 2.6.2 of FPC. Researching on the web for ways of compiling this version it seems that installation steps vary depending on the version. You are more experienced in Pascal than me, so do you know if your code can easily modified to build version I need?

I can do that, with some effort, but I'd like to know the effort.

**Comment by kveroneau** [comment]
Yes, the Dockerfile should be-able to work with any version of the FPC compiler.  FPC is actually really easy to install once you know the steps.  For docker, you will need to update the FPC 2.6.2 install.sh file to remove it's interactive acting of questions, but if you know just a little bit of Bash, it's really easy to replace those lines with a hardcoded variable.  Do a diff on my modified install.sh with the official one if you need to see what changes were made.  Then in the Dockerfile replace the references of the version of FPC I am using with FPC 2.6.2 and it should work without a hitch.
